samples/tableadmin.js

```diff
@@ -0,0 +1,289 @@
+        return;
+    };
+    // [START bigtable_create_family_gc_max_age]
+
+    console.log('\nCreating column family cf2 with max versions GC rule...');
+    // [START bigtable_create_family_gc_max_versions]
+    // Create a column family with GC policy : most recent N versions
+    // where 1 = most recent version
+
+    // Define the GC policy to retain only the most recent 2 versions
+    const maxVersionsRule = {
+        rule: {
+            versions: 2
+        }
+    };
+
+    // Create a column family with given GC rule
+    try {
+        const [family, apiResponse] = await table.createFamily('cf2', maxVersionsRule);
+        console.log(`Created column family ${family.id}`);
+    } catch (err) {
+        console.error(`Error creating column family ${err}`);
+        return;
+    };
+    // [END bigtable_create_family_gc_max_versions]
+
+    console.log('\nCreating column family cf3 with union GC rule...');
+    // [START bigtable_create_family_gc_union]
+    // Create a column family with GC policy to drop data that matches atleast one condition.
+
+    // Define a GC rule to drop cells older than 5 days or not the most recent version
+    const unionRule = {
+        rule: {
+            versions: 1,
+            age: {
+                seconds: 432000,
+                nanos: 0
+            },
+            union: true
+        }
+    };
+
+    try {
+        const [family, apiResponse] = await table.createFamily('cf3', unionRule);
+        console.log(`Created column family ${family.id}`);
+    } catch (err) {
+        console.error(`Error creating column family ${err}`);
+        return;
+    };
+    // [END bigtable_create_family_gc_union]
+
+    console.log('\nCreating column family cf4 with intersect GC rule...');
+    // [START bigtable_create_family_gc_intersection]
+    // Create a column family with GC policy to drop data that matches all conditions
+
+    // GC rule: Drop cells older than 5 days AND older than the most recent 2 versions
+    const intersectionRule = {
+        rule: {
+            versions: 2,
+            age: {
+                seconds: 432000,
+                nanos: 0
+            },
+            intersection: true
+        }
+    }
+    try {
+        const [family, apiResponse] = await table.createFamily('cf4', intersectionRule);
+        console.log(`Created column family ${family.id}`);
+    } catch (err) {
+        console.error(`Error creating column family ${err}`);
+        return;
+    };
+    // [END bigtable_create_family_gc_intersection]
+
+    console.log('\nPrinting ID and GC Rule for all column families...');
+    // [START bigtable_list_column_families]
+    // List all families in the table with GC rules
+    try {
+        const [families, apiResponse] = await table.getFamilies();
+        // Print ID, GC Rule for each column family
+        families.forEach(function (family) {
+            console.log(`Column family: ${family.id}, Metadata: ${JSON.stringify(family.metadata)}`);
```

Comment:
For complex logging expressions it may be helpful to comment an actual output of the line